### PR TITLE
Prompt with replay file

### PR DESCRIPTION
**Pull request by w1ndblow**
if replayfile have not all answers relatively cookiecutter config file it'll execute prompt operation

### CODE REVIEW

**Comment by w1ndblow** [comment]
hi, @audreyfeldroy, @browniebroke, @ericof, @hackebrot, @insspb, @pydanny, @simobasso, @ssbarnea could you take a look my PR, please

File: test_requirements.txt [hunk 1]
```diff
@@ -4,3 +4,4 @@ pytest-mock
 freezegun
 safety
 pre-commit
+defer
```
Comment (kurtmckee) [inline test_requirements.txt:7]: Where is this used?

**Review by kurtmckee — CHANGES_REQUESTED** [review]
@w1ndblow Thanks for your work on this.

Would you add tests that demonstrate that this feature is working as expected?

File: test_requirements.txt (hunk 1 above)
Comment (w1ndblow) [reply test_requirements.txt]: missclick

**Comment by w1ndblow** [comment]
i added one test, i can add more if needed

**Comment by w1ndblow** [comment]
hi, could any one merge it? i merged this with main 3 times already :)

**Review by ericof — APPROVED** [review]

**Pull request merged**